packages/perseus/src/widgets/matrix/matrix.tsx

```diff
@@ -537,16 +533,27 @@ class Matrix extends React.Component<Props, State> implements Widget {
     }
 }
 
-const propTransform: (arg1: any) => any = (editorProps) => {
+// NOTE: Answers here in RenderProps does not contain answers from the rubric.
+// It's an empty array of the same length and depth used to render the matrix.
+type RenderProps = MatrixPublicWidgetOptions & {
+    answers: ReadonlyArray<ReadonlyArray<string>>;
+};
+
+function transform(widgetOptions: MatrixPublicWidgetOptions): RenderProps {
     // Remove answers before passing to widget
-    const blankAnswers = _(editorProps.matrixBoardSize[0]).times(function () {
-        return stringArrayOfSize(editorProps.matrixBoardSize[1]);
+    const blankAnswers = _(widgetOptions.matrixBoardSize[0]).times(function () {
```

Comment:
I made a helper for this in my PR: https://github.com/Khan/perseus/pull/2396/files#diff-5d41841bf6867bd9e265d0ff7631b205e25c0d4ea16386b1bce446e324724575

You could rebase on top of mine or wait until I land into `main` (maybe tomorrow since I don't understand what's happening with releases right now)

Reply:
Updated it to use your helper. :)